### ISSUE TITLE
Maintenance support

### ISSUE THREAD

**Issue by RobWin** [Swagger2Markup/swagger2markup#312]
Dear community,

unfortunately I'm very busy at the moment and can't maintain Swagger2Markup alone anymore.
There are many interesting new topics:

1. Swagger v3 support
2. Merge Swagger2Markup repositories and create a new multi-module repository. It doesn't matter if we use Gradle or Maven to build the project. Any help is welcome.

Is someone interested to help me? 

Kind regards,
Robert

**Comment by austek** [comment]
I might be able to help.

What's the plan? I think, it'll be better to co-ordinate and then carry on

**Comment by RobWin** [comment]
Hi, 

you could help by creating some PR (OpenAPI v3 support?) or reply to issues/questions.
I do have a Slack channel.

**Comment by austek** [comment]
Hi, 
Could you invite me on slack pls

**Comment by RobWin** [comment]
done

**Comment by tntclaus** [comment]
> Merge Swagger2Markup repositories and create a new multi-module repository. It doesn't matter if we use Gradle or Maven to build the project. Any help is welcome.

@RobWin is this still an issue? Probably I can do that. Do you have any vision which repositories (if not all of them) should be merged in which order?

**Comment by RobWin** [comment]
In the master branch we already merged a lot of modules. But we still need help to maintain Swagger2Markup,

**Comment by tntclaus** [comment]
@RobWin  yes, add me please. I can't promise I will do a lot, but I will do what I can.

**Comment by ewoks** [comment]
Any progress or status update you could share @tntclaus ?

**Comment by snowe2010** [comment]
It's been several years since updates to any of these threads. Is there any progress on this or is this project dead in the water?

**Comment by RobWin** [comment]
Hi, @ppatierno was so kind to take over the project.

**Comment by ppatierno** [comment]
@ShubhamRwt is taking a look at moving to use GitHub actions and Sonatype for next release.

**Comment by snowe2010** [comment]
Thanks for the update. Is there any progress on the v3 support or is that a reasonably long way off? Need to judge whether we should begin using this project yet or delay.
